components/ConfigureQuiz/constants.js

```diff
@@ -0,0 +1 @@
+export const DIFFICULTY_OPTIONS = [{ label: 'easy', value: 'easy' }, { label: 'medium', value: 'medium' }, { label: 'hard', value: 'hard' }];
```

Comment:
I prefer a separate constants folder at src, with the constants stored in their own sub folders so that they can be seen quickly. With this structure, you have to go to each individual component and open its containing folder to see if any constants apply to it, whereas under src/constants/comp folder name you'd see it immediately. I'd recommend that refactor but it's not critical, I'll leave it up to you.

Reply:
Okay, got it. I'll do some thinking and see how I can restructure it. The whole concept of importing/exporting stuff separately is super brand new to me still (i.e. as of this project) so this is just something Sean recommended for me. I've been meaning to look at where I can pull more things out into their own files but haven't gotten to that yet, I'll see if I can refactor this on the next go-around.